dockerunit-core/src/main/java/com/github/qzagarese/dockerunit/annotation/impl/PortBindingWrapperExtensionInterpreter.java

```diff
@@ -4,14 +4,14 @@
 import com.github.qzagarese.dockerunit.annotation.ExtensionInterpreter;
 import com.github.qzagarese.dockerunit.annotation.PortBinding;
 import com.github.qzagarese.dockerunit.annotation.PortBindings;
-import com.github.qzagarese.dockerunit.internal.TestDescriptor;
+import com.github.qzagarese.dockerunit.internal.ServiceDescriptor;
 
 public class PortBindingWrapperExtensionInterpreter implements ExtensionInterpreter<PortBindings>{
 
     private PortBindingExtensionInterpreter builder = new PortBindingExtensionInterpreter();
     
     @Override
-    public CreateContainerCmd build(TestDescriptor td, CreateContainerCmd cmd, PortBindings pbs) {
+    public CreateContainerCmd build(ServiceDescriptor td, CreateContainerCmd cmd, PortBindings pbs) {
```

Comment:
you changed the type but now the variable name seems out of place (what does `td` mean seeing that that type is `ServiceDescriptor` ?)

Reply:
this is the result of a refactoring, thank you!